site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -96,6 +96,9 @@ EOF
 # Add ca.crt to the allocator-tls-ca Secret
 TLS_CA_VALUE=$(kubectl get secret allocator-tls -n agones-system -ojsonpath='{.data.ca\.crt}')
 kubectl get secret allocator-tls-ca -o json -n agones-system | jq '.data["tls-ca.crt"]="'${TLS_CA_VALUE}'"' | kubectl apply -f -
+echo $TLS_CA_VALUE | base64 -d > ca.crt
```

Comment:
Are these changes only necessary once we make the next release (and are using go 1.15)? If so, they should be behind a feature gate: https://agones.dev/site/docs/contribute/#within-a-page

Reply:
I don't think they need to go behind a feature gate, as they will still work with the current release - and if people do it now, they don't have to do it later. So no issue as it is on my end.

Reply:
as long as they work with both old and new versions, sgtm.

Reply:
Go 1.15 no longer supports Common Name based certs for HTTPS without a specific flag.  Adding SAN to the cert is the solution and it's also backward compatible.